core/libraries/plugin_api/EE_Register_Addon.lib.php

```diff
@@ -375,114 +396,114 @@ private static function _get_addon_settings($class_name, array $setup_args)
             'class_name'            => $class_name,
             // the addon slug for use in URLs, etc
             'plugin_slug'           => isset($setup_args['plugin_slug'])
-                ? (string) $setup_args['plugin_slug']
+                ? (string)$setup_args['plugin_slug']
```

Comment:
existing code was correct here as well as we are adding a space after type casts

Reply:
again, weird that happened, but ok